docs/getting-started/sdk/index.md

```diff
@@ -38,7 +42,56 @@ To build the SDK, run the following command:
 cargo build
 ```
 
-For more information on how to use the SDK, see the [repository](https://github.com/bitwarden/sdk).
+## Linking the SDK to clients
+
+After modifying the SDK, it can be beneficial to test the changes in the client applications. To do
+so you will need to update the SDK reference in the client applications.
+
+These instructions assumes you have a directory structure similar to:
+
+```text
+sdk/
+clients/
+ios/
+android/
+```
+
+### Web clients
+
+The web clients uses NPM to install the SDK as a dependency. NPM offers a dedicated command
+[`link`][npm-link] which can be used to temporarily replace the packages with a local version.
+
+```bash
+npm link ../sdk/languages/sdk-client ../sdk/languages/wasm
+```
+
+:::warning
+
+Running `npm ci` or `npm install` will replace the linked packages with the published version.
+
+:::
+
+### Mobile
+
+#### Android
+
+1. Build and publish the SDK to the local Maven repository:
+
+   ```bash
+   ../sdk/languages/kotlin/publish-local.sh
+   ```
```

Comment:
Whitespaces?

```suggestion
```bash
../sdk/languages/kotlin/publish-local.sh
```
```

Reply:
This is intentional as the block appears inside a list. The indention ensures we don't escape the list item.